### ISSUE TITLE
Help page for OORT command line

### ISSUE THREAD

**Issue by rlwalls2008** [owlcollab/owltools#116]
Not to complain too much, but the page https://github.com/owlcollab/owltools/wiki/Oort-Command-Line does not actually provide any instructions on how to run OORT on the command sweetie. The help file alone (at owltools -h) is not sufficient for me to know how to run the ontology release manager. 

I installed OWLtools from the jars on Jenkins, and tried to use "ontology-release-runner -h", but I got the message "command not found".

I also tried "owltools ontology-release-runner -h" but got "could not parse:ontology-release-runner" followed by a long error message.

I know the wiki is in transition. Is there any old documentation  you can point me to to get started? I saw this (https://github.com/owlcollab/owltools/wiki/Command-Line-Examples) page, but it was a bit high level.

Do I need to download "ontology-release-runner" from Jenkins as well?

**Comment by rlwalls2008** [comment]
I made some progress by downloading ontology-release-runner and owltools-oort-all.jar and placing them both in /bin. I can now see the OORT help file.

**Comment by hdietze** [comment]
Yes, the OORT documentation, is out-dated and incomplete. It is on my TODO list to update that.

**Comment by cmungall** [comment]
We're not planning to spend too much time updating the OORT docs as this will be superseded by ROBOT (Robot is an OBO Tool) being developed by @jamesaoverton https://github.com/ontodev/owltools2-experimental

...but we want to make sure people aren't completely flummoxed in the interim, so yes we at least need better info on getting started

**Comment by rlwalls2008** [comment]
I think what would really help is a few command line examples of how to do a release. Even though the help file says to use:

"bin/ontology-release-runner [OPTIONAL OPTIONS] ONTOLOGIES-FILES"

and lists the options, in my 20 minutes of playing around with it, I couldn't find a functional syntax.

**Comment by hdietze** [comment]
If it helps, you could try the graphical user interface (GUI) for OORT: Download the [ontology-release-runner-gui](http://build.berkeleybop.org/userContent/owltools/ontology-release-runner-gui) script. This script uses also the owltools-oort-all.jar.

**Comment by rlwalls2008** [comment]
I did think of that and download that script, but two problems:
1. (very simple) I can't figure out what command to use to open the GUI (I am a java novice)
2. In the current (older) version of the GUI I have, I cannot figure out how to use the force release option to let me include equivalent classes in my release (nor do I particularly understand why that is an error). Maybe it is obvious in the new version.

**Comment by fbastian** [comment]
Actually, I remember that I couldn't find some perl scripts used by the Makefile provided, to generate, e.g., the diffs between releases. I think I found them on an obscure repository belonging to Chris. Providing all the perl scripts would help (or at least, pointing to their location).

**Comment by hdietze** [comment]
The ontology-release-runner-gui script is a simple bash script wrapping the java command. You can just open it in a text editor to see the details.  To run the script, you might need to make it executable using the `chmod` command:
```
chmod 755 ontology-release-runner-gui
```

And you are right, the current OORT GUI does not have a flag to force the release. In theory you can add that flag during the startup of the GUI  `--force`. That would just look like:
```
./ontology-release-runner-gui --force
```

**Comment by cmungall** [comment]
They are here:

https://github.com/cmungall/obo-scripts

They are concentrated pure evil. Don't touch them.

I warned you....

Seriously whilst there are various Makefiles kicking around that use these their use should be phased out

**Comment by rlwalls2008** [comment]
Yes, that flag for force release worked, Heiko. Thanks! Also, I didn't realize I had to be in the same directory to get the GUI to run. Thought I had it in my PATH, but it doesn't seem respond the same way as ontology-release-runner (which I can run from wherever).

**Comment by fbastian** [comment]
Haha Chris, these scripts are evil, but very useful! What do you currently use for releasing ontologies, if not these scripts?

**Comment by cmungall** [comment]
Most ontologies use only Makefile-driven owltools or oort.

 * https://github.com/obophenotype/cephalopod-ontology/blob/master/src/ontology/Makefile
 * https://github.com/obophenotype/porifiera-ontology/blob/master/src/ontology/Makefile
 * https://github.com/EnvironmentOntology/envo/blob/master/src/envo/Makefile
 * http://viewvc.geneontology.org/viewvc/GO-SVN/trunk/ontology/extensions/Makefile?view=markup

Some like GO have an extra layer of checks that happen downstream with hacky perl scripts like this:
http://viewvc.geneontology.org/viewvc/GO-SVN/trunk/software/utilities/check-obo-for-standard-release.pl?view=markup

Uberon is a big mix, but we hope to standardize with ROBOT
